docs/installation.md

```diff
@@ -39,7 +39,23 @@
        - { resource: "@SyliusPayPalPlugin/config/config.yaml" }
    ```
 
-4. Apply migrations
+1. Add `FOS\RestBundle` configuration to your `config/packages/fos_rest.yaml` file
```

Comment:
Since this plugin requires it, I would be in favour of adding this directly to the plugin's configuration, to reduce number of installation steps